schema-build/compile-and-run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+WORKDIR="./target"
+echo ""
+echo "Downloading the cwl-avro.yml file..."
+echo ""
+curl -o ${WORKDIR}/CommandLineTool.yml https://raw.githubusercontent.com/common-workflow-language/common-workflow-language/master/draft-3/CommandLineTool.yml
+curl -o ${WORKDIR}/Process.yml https://raw.githubusercontent.com/common-workflow-language/common-workflow-language/master/draft-3/Process.yml
+curl -o ${WORKDIR}/Workflow.yml https://raw.githubusercontent.com/common-workflow-language/common-workflow-language/master/draft-3/Workflow.yml
+
+echo "Downloading the metaschema.yml file..."
+echo ""
+curl -o ${WORKDIR}/metaschema.yml https://raw.githubusercontent.com/common-workflow-language/schema_salad/master/schema_salad/metaschema.yml
+
+echo "Fixing the cwl-avro.yml file for OutputRecordField..."
+echo ""
+sed "921s/  extends: \"sld:RecordSchema\"/  extends: \"sld:RecordField\"/" ${WORKDIR}/CommandLineTool.yml  ${WORKDIR}/Process.yml ${WORKDIR}/Workflow.yml > ${WORKDIR}/cwl-avro-fixed-OutputRecordField.yml
```

Comment:
As things shifted in the updated draft, the location and line-number changed and this will fix it:

Before:

```Bash
sed "921s/  extends: \"sld:RecordSchema\"/  extends: \"sld:RecordField\"/" ${WORKDIR}/CommandLineTool.yml  ${WORKDIR}/Process.yml ${WORKDIR}/Workflow.yml > ${WORKDIR}/cwl-avro-fixed-OutputRecordField.yml
```

After:

```Bash
sed "286s/  extends: \"sld:RecordSchema\"/  extends: \"sld:RecordField\"/" ${WORKDIR}/Process.yml > ${WORKDIR}/Process-fixed-OutputRecordField.yml
```